test/common/matchers.dart

```diff
@@ -25,7 +27,7 @@ class _AssetMatcher extends Matcher {
   const _AssetMatcher(this._expected);
 
   @override
-  bool matches(item, _) =>
+  bool matches(Object item, _) =>
```

Comment:
Hrm – trying to make the linter happy. I really don't like this guideline at all.